index.d.ts

```diff
@@ -51,6 +58,18 @@ camelCase(['foo', 'bar']);
 
 camelCase(['__foo__', '--bar'], {pascalCase: true});
 //=> 'FooBar'
+
+camelCase('lorem-ipsum', {locale: 'en-US'});
+//=> 'loremIpsum'
```

Comment:
It's enough with this example here. The this and the other examples can be in the doc comment for the option.

Reply:
Kept single example here. All examples are in doc comment for `locale` option.

Reply:
> All examples are in doc comment for locale option.

I think you forgot to add it there. Also add those to the readme (`locale` option part).

Reply:
Sorry, I forgot. 
There are 4 examples in total. All goes under 'Usage' section or 'API' section? I think 'API' section is appropriate as all examples are similar format (`camelcase(....)`).

Reply:
So, I guess this is also resolved.